### PR TITLE
some minor changes & additions

### PR DESCRIPTION
**Pull request by dsager**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
-# dependencies specified in in appsignal_report.gemspec
```
Comment (dsager) [inline Gemfile:3]: :see_no_evil:

**Comment by nicolas-fricke** [comment]
+1

**Comment by miguelbernadi** [comment]
+1

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -31,6 +31,13 @@
     mocks.verify_partial_doubles = true
   end
 
+  # This allows you to limit a spec run to individual examples or groups
+  # you care about by tagging them with `:focus` metadata. When nothing
+  # is tagged with `:focus`, all examples get run. RSpec also provides
+  # aliases for `it`, `describe`, and `context` that include `:focus`
+  # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
+  config.filter_run_when_matching :focus
```
Comment (yaneq) [inline spec/spec_helper.rb:39]: 👏

**Comment by yaneq** [comment]
+1

**Pull request merged**